content/applications/websites/ecommerce.rst

```diff
@@ -24,5 +24,5 @@ products and increase your average cart sizes.
    :titlesonly:
 
    ecommerce/products
-   ecommerce/checkout_payment_shipping
+   ecommerce/checkout_payment_delivery
```

Comment:
This requires redirect rules.

Reply:
And, since you'll write one for `shipping.rst` too, it's the chance to rename the file and anchors to "delivery".

Reply:
![both-agree](https://github.com/user-attachments/assets/e256bd50-c7f4-4f55-82b4-c776f86bf524)